src/webviews/apps/home/components/preview-banner.ts

```diff
@@ -95,26 +98,10 @@ export class GlPreviewBanner extends LitElement {
 
 		return html`
 			<gl-card>
-				<p><strong>Welcome to the ${this.isNewInstall ? 'GitLens ' : 'new '}Home View!</strong></p>
-				<p>
-					${this.isNewInstall
-						? html`This is a hub for your current, future, and recent work. `
-						: html`We've reimagined GitLens' Home to be a more helpful daily workflow tool. `}We're
-					continuing to refine this experience and welcome your
-					<a class="feedback" href="https://github.com/gitkraken/vscode-gitlens/discussions/3721"
-						><code-icon icon="feedback"></code-icon> feedback</a
-					>.
-				</p>
-				${when(
-					!this.isNewInstall,
-					() => html`
-						<button-container>
-							<gl-button appearance="secondary" @click=${() => this.togglePreview(true)} full
-								><code-icon icon="arrow-left"></code-icon> Revert to Old Home View</gl-button
-							>
-						</button-container>
-					`,
-				)}
+				<h4 class="title">Welcome to the GitLens Home View</h4>
+				Focus, prioritize, and take action on your current work, what’s next, and what you’ve recently worked
```

Comment:
@nzaytsev please update this message to "Streamline your workflow—effortlessly track, manage, and collaborate on your branches and pull requests—all in one intuitive hub."

Reply:
This is what you expect?
![Screenshot 2024-12-19 at 09 52 24](https://github.com/user-attachments/assets/4c4d20da-26b5-4527-aa8d-933885fde3d6)